src/App.tsx

```diff
@@ -1,26 +1,47 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+  .map((n: number) => `Item ${n}`);
+
+function getCurrentItems(currentPage:number, perPage: number): string[] {
+  if (currentPage === 1) {
+    return [...items].splice(0, perPage);
```

Comment:
it's better to use `slice` in such cases - it doesn't mutate the original array, so no need to make a copy